lib/main_common.pm

```diff
@@ -2080,7 +2080,7 @@ sub load_security_tests_crypt_tool {
     load_security_console_prepare;
 
     loadtest "console/gpg";
-    loadtest "console/yast2_dm_crypt";
```

Comment:
it was the only place where it was called so better to delete it in same commit. I have idea why you getting this conflict. maybe someone introduced changes **after** you branch from master. can you please rebase your branch to latest master and try to delete this file again ?